manager/manifests/prometheus-statsd-exporter.yaml

```diff
@@ -23,17 +23,13 @@ data:
       observer_type: histogram
 
 ---
-
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
```

Comment:
`_cortexCPUReserve` and `_cortexMemReserve` should be updated accordingly (see the comment above their definitions). It's also worth checking that `cortex cluster info` displays the right info for capacity.

Reply:
Good thinking, updated.